console/atest-ui/src/views/net.ts

```diff
@@ -533,10 +533,26 @@ function getToken() {
   return token
 }
 
+function GetYaml(suite: string, store: string,callback: (d: any) => void, errHandle?: (e: any) => void | null){
```

Comment:
imo, we can use the arrow function in ts, which avoids a lot of other problems .🤔
cc @LinuxSuRen

Reply:
Please feel free to give us an example.

Reply:
It is very obvious that arrow functions avoid the problem of this pointing, function type functions are determined at the time they are called, arrow functions bind this according to the current context, with the release of es6 we should use arrow functions instead of the.

Reply:
I got you now. My JavaScript knowledge still comes from ten years ago.

https://www.w3schools.com/js/js_arrow_function.asp

Reply:
> It is very obvious that arrow functions avoid the problem of this pointing, function type functions are determined at the time they are called, arrow functions bind this according to the current context, with the release of es6 we should use arrow functions instead of the.

thank you,I'll fix it.